python-package/lightgbm/basic.py

```diff
@@ -1245,12 +1245,12 @@ def __getstate__(self):
         return this
 
     def __setstate__(self, state):
-        model_str = state.get('handle', None)
+        model_str = state.get('handle', None)  # list of str
```

Comment:
Really?
I don't think so because of `f.writelines(model)`.
(note: writelines takes lsit of str)